proto/v1/sql_service.proto

```diff
@@ -19,6 +20,12 @@ service SQLService {
       body: "*"
     };
   }
+  rpc ListQueryHistory(ListQueryHistoryRequest) returns (ListQueryHistoryResponse) {
+    option (google.api.http) = {
+      get: "/v1/{name=instances/*/databases/*}/queryHistory"
```

Comment:
This is similar to worksheets that is personal.

Reply:
You mean just `get: "/v1/queryHistory"`? Then we may need the `filter` field (instance = xxx/database = xxx) in `ListQueryHistoryRequest` to filter data in the database or instance to reduce request cost🤔

Reply:
Or we can support
```
option (google.api.http) = {
      get: "/v1/queryHistory"
      additional_bindings {get: "/v1/{name=instances/*/databases/*}/queryHistory"}
      additional_bindings {get: "/v1/{name=instances/*}/queryHistory"}
};
```

Reply:
We can start with filter with database (instances/x/databases/y) only, and this is optional.